Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -409,6 +411,40 @@ ModelExperimentalSceneGraph.prototype.updateModelMatrix = function () {
   }
 };
 
+function forEachRuntimePrimitive(sceneGraph, callback) {
+  for (let i = 0; i < sceneGraph._runtimeNodes.length; i++) {
+    const runtimeNode = sceneGraph._runtimeNodes[i];
+    for (let j = 0; j < runtimeNode.runtimePrimitives.length; j++) {
+      const runtimePrimitive = runtimeNode.runtimePrimitives[j];
+      callback(runtimePrimitive);
+    }
+  }
+}
+
+/**
+ * Traverses through all draw commands and changes the back-face cull setting.
+ *
+ * @param {Boolean} backFaceCulling The new value for the back-face cull setting.
```

Comment:
Just a tiny wording tweak
```suggestion
 * Traverses through all draw commands and changes the back-face culling setting.
 *
 * @param {Boolean} backFaceCulling The new value for the back-face culling setting.
```